tools/sdk-lints/src/main.rs

```diff
@@ -34,6 +69,18 @@ fn clap_app() -> App<'static, 'static> {
                         .required(false)
                         .long("readme"),
                 )
+                .arg(
+                    Arg::with_name("cargotoml")
```

Comment:
Do you know about structopt? https://lib.rs/crates/structopt
I find it results in more readable CLI configurations.

Reply:
yeah all of our other CLIs use Clap directly so I was aiming for consistency. We can discuss as a team if we want to go one way or the other